test/gleam/float_test.gleam

```diff
@@ -390,3 +390,51 @@ pub fn divide_test() {
   float.divide(1.0, by: 0.0)
   |> should.equal(Error(Nil))
 }
+
+pub fn add_test() {
+  float.add(1.0, 2.0)
+  |> should.equal(3.0)
+
+  list.fold([1.0, 2.0, 3.0], 0.0, float.add)
```

Comment:
I would put list.fold calls into list_test.gleam into fold_test

Reply:
I think we can remove these `list.fold` test cases, I don't think it adds anything other the tests without `list.fold`.